sklearn/metrics/tests/test_regression.py

```diff
@@ -207,13 +217,15 @@ def test_regression_multioutput_array():
 
     mse = mean_squared_error(y_true, y_pred, multioutput='raw_values')
     mae = mean_absolute_error(y_true, y_pred, multioutput='raw_values')
+    maq = pinball_error(y_true, y_pred, multioutput='raw_values')
```

Comment:
maq == "mean absolute quantile"? It does not standard to me. Maybe just `pbl` for pinball loss.